packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs/LineGraph/Reflow/DynamicData.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import LineGraph from '@cerner/terra-graphs/lib/components/Line/LineGraph';
+import Button from 'terra-button/lib/Button';
+import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
+import '@cerner/terra-graphs-docs/lib/Css/ExampleGraphContainer.module.scss';
+import getConfigLineTimeseriesPanning from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/Line/lineTimeseriesPanning';
+import data from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Line/panningData';
+
+/*
+Please refer to the documentation below to see the graphConfig and data objects
+*/
+let graphConfig = utils.deepClone(getConfigLineTimeseriesPanning('#linePanningExample'));
+const dataset = [utils.deepClone(data[0])];
+const newDataset = {};
+// let dataset = utils.deepClone(data);
```

Comment:
Remove comment here if not needed.

Reply:
fixed here: 595e44c7eaf30f3e0cc3212e7ad7c4a7186e25e2